client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1084,6 +1084,11 @@ Undocumented.prototype.readFeed = function( query, fn ) {
 	this.wpcom.req.get( '/read/feed/' + encodeURIComponent( query.ID ), params, fn );
 };
 
+Undocumented.prototype.discoverFeed = function( query, fn ) {
+	debug( '/read/feed' );
+	this.wpcom.req.get( '/read/feed/', query, fn );
```

Comment:
even though you're not using it, return the Promise that comes out of the get call. Might need it later.